spec/app_perf_rpm_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'socket'
```

Comment:
I was receiving this error running just that spec: 

```
hbeaver~/code/vendor/app_perf_rpm (improve_testing)$ be rspec spec/app_perf_rpm_spec.rb

Randomized with seed 12788
.F...

Failures:

  1) AppPerfRpm#host returns the host name
     Failure/Error: expect(Socket).to receive(:gethostname) { "host" }

     NameError:
       uninitialized constant Socket
     # ./spec/app_perf_rpm_spec.rb:10:in `block (3 levels) in <top (required)>'

Finished in 0.01044 seconds (files took 0.16756 seconds to load)
5 examples, 1 failure

Failed examples:

rspec ./spec/app_perf_rpm_spec.rb:9 # AppPerfRpm#host returns the host name

Randomized with seed 12788
```

It doesn't appear to be order dependent for specs when executing suite (which is a reason I added `:random` ordering to RSpec config. 

My current system:
Ruby 2.2.2p95
MaxOS 10.12.4